packages/table/src/table.tsx

```diff
@@ -1004,6 +1092,53 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.handleSelection([]);
     }
 
+    private handleFocusMove = (e: KeyboardEvent, direction: "up" | "down" | "left" | "right") => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const {focusedCellCoordinates} = this.state;
+        const newFocusedCellCoordinates = { col: focusedCellCoordinates.col, row: focusedCellCoordinates.row };
+        const { grid } = this;
+        if (direction === "up") {
+            newFocusedCellCoordinates.row--;
+        }
+        if (direction === "down") {
+            newFocusedCellCoordinates.row++;
+        }
+        if (direction === "left") {
+            newFocusedCellCoordinates.col--;
+        }
+        if (direction === "right") {
+            newFocusedCellCoordinates.col++;
+        }
+        if (newFocusedCellCoordinates.row < 0 || newFocusedCellCoordinates.row >= grid.numRows ||
+            newFocusedCellCoordinates.col < 0 || newFocusedCellCoordinates.col >= grid.numCols) {
+            return;
```

Comment:
what are the implications of not invoking the callbacks here? do we want to notify listeners that focus was removed?